go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/containers/image/v5 v5.9.0
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/go-logr/logr v0.3.0
+	github.com/google/go-licenses v0.0.0-20210727195037-24e2421190d8 // indirect
```

Comment:
Did you add this on purpose? 
What is it used for?

Reply:
That has probably been added by the deploy_local script, since it installs go-licenses, which generates the third-party licenses of the libraries we use. Those licenses are then expected by our Dockerfile and bundled with the image, before it is pushed to quay